tests/test_patch_rebaser.py

```diff
@@ -322,3 +323,59 @@ def test_packages_to_process_skips_packages_not_in_the_list(
         main()
 
     mock_gitrepo.assert_not_called()
+
+
+def test_rebase_exception_gitreview(mock_repo):
```

Comment:
Thank you for adding these tests!  If you don't mind, one more minor request on them.  If you look at the others that @jpichon has added we have a little docstring block at the top which explains the expected behavior under test (in 'Given, When, Then' format).  If you could, please add this, it makes it much easier for others to come in and work with the code, as they can more quickly understand what it is attempting to do.

Reply:
Done, thanks for the review!